### ISSUE TITLE
Accessibility: Pagination links cannot receive focus

### ISSUE THREAD

**Issue by mdarens** [AZaviruha/pager#3]
This pager currently doesn't add `href` or `tabIndex` to its links, which makes it substantially more difficult for users to navigate pages with the keyboard.
# Proposed Solution

Add `href`s to the links in the `render` method of the `Page` class and update the handlers to call `preventDefault` on the event.
